dockerplugin.py

```diff
@@ -611,15 +701,39 @@ def read_callback(self):
                 if not read_at:
                     # No stats available yet; skipping container.
                     continue
-
+                # Get cgroup info container by inspecting the container
+                container_inspect = self.client.inspect_container(container)
+                cpu_shares = container_inspect['HostConfig']['CpuShares']
+                # Calculate the sum of weights of all containers
+                sum_of_shares += (1024 if cpu_shares == 0 else cpu_shares)
```

Comment:
Looks like you're using this default of 1024 multiple times. Should be a constant.